projects/bp-gallery/src/components/views/exhibitions/ExhibitionTool.tsx

```diff
@@ -0,0 +1,351 @@
+import { ExpandLess, ExpandMore } from '@mui/icons-material';
+import { Button, IconButton, Paper, TextField } from '@mui/material';
+import { useContext, useEffect, useRef, useState } from 'react';
+import { useGetExhibitionQuery } from '../../../graphql/APIConnector';
+import { useSimplifiedQueryResponseData } from '../../../graphql/queryUtils';
+import { FlatExhibition } from '../../../types/additionalFlatTypes';
+import TextEditor from '../../common/editors/TextEditor';
+import { useTranslation } from 'react-i18next';
+import {
+  ExhibitionGetContext,
+  ExhibitionSectionUtilsContext,
+  ExhibitionSetContext,
+  ExhibitionStateManager,
+} from './ExhibitonUtils';
+import {
+  DndContext,
+  DragOverlay,
+  DragEndEvent,
+  DragStartEvent,
+  UniqueIdentifier,
+  useDroppable,
+} from '@dnd-kit/core';
+import { SortableContext } from '@dnd-kit/sortable';
+import { channelFactory } from '../../../helpers/channel-helpers';
+
+const IdeaLot = () => {
+  const idealot = useContext(ExhibitionGetContext).getIdealot();
```

Comment:
probably rename this to ideaLot everywhere? You sometimes write the second part in uppercase and sometimes not

Reply:
It's idealot now everywhere